single-line-diagram/single-line-diagram-core/src/main/resources/animations.css

```diff
@@ -31,3 +31,57 @@
 .sld-arrow-animation-low-speed { --sld-arrow-animation-parameters: 3s infinite linear }
 .sld-arrow-animation-average-speed { --sld-arrow-animation-parameters: 2s infinite linear }
 .sld-arrow-animation-high-speed { --sld-arrow-animation-parameters: 1s infinite linear }
+
+:root {
+  --sld-overload-color: #ffeb3b; /* Yellow for overload */
+  --sld-overvoltage-color: #5a0101; /* Red for overvoltage */
```

Comment:
this is very brown, and very close to the "fictitious" color. Besides, we should try to have the same colour as on network-area-diagram side.